### PR TITLE
Deadline extension to May 22

### PR DESCRIPTION
**Pull request by hauschke**

### CODE REVIEW

File: talk-proposals.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Your presentation or lightning talk may take the form of a demonstration (screen
 
 ##### Proposal and review process
 
-1. **Before May 15, 2020:** [Share your proposal through OpenReview](https://openreview.net/group?id=vivoconference.org/VIVO/2020/Conference), where it will be publicly visible. The proposal should be submitted as an abstract of 150 - 350 words.
+1. ~~**Before May 15, 2020:**~~ **EXTENDED DEADLINE: Before May, 2020** [Share your proposal through OpenReview](https://openreview.net/group?id=vivoconference.org/VIVO/2020/Conference), where it will be publicly visible. The proposal should be submitted as an abstract of 150 - 350 words.
```
Comment (awoods) [inline talk-proposals.md:23]: It is not clear from this update what the new deadline is.

**Comment by hauschke** [comment]
That happens when something is being done too quick. But now it's done, I hope. Thanks!

File: talk-proposals.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ Your presentation or lightning talk may take the form of a demonstration (screen
 
 ##### Proposal and review process
 
-1. ~~**Before May 15, 2020:**~~ **EXTENDED DEADLINE: Before May, 2020** [Share your proposal through OpenReview](https://openreview.net/group?id=vivoconference.org/VIVO/2020/Conference), where it will be publicly visible. The proposal should be submitted as an abstract of 150 - 350 words.
+1. ~~**Before May 15, 2020:**~~ **EXTENDED DEADLINE: Before May 22, 2020** [Share your proposal through OpenReview](https://openreview.net/group?id=vivoconference.org/VIVO/2020/Conference), where it will be publicly visible. The proposal should be submitted as an abstract of 150 - 350 words.
 2. Share the OpenReview link of your proposal on social media and your networks, and invite people to comment and engage.
 3. Keep an eye on the interactions, and reply when meaningful.
 4. **Between the date of submission and May 22:** Assigned reviewers start posting reviews from 15 May until May 31; they will become visible immediately and you can react to them.
```
Comment (awoods) [inline talk-proposals.md:26]: Should the dates in this bullet also be updated?

Comment (mconlon17) [reply talk-proposals.md:26]: No.  We'll make due as best we can.

**Pull request merged**